beta/kustomize/overlays/self-managed-tls/README.md

```diff
@@ -0,0 +1,5 @@
+# Self-Managed TLS
+
+If you'd rather bring your own TLS certificate, this overlay offers you that ability.
```

Comment:
Perhaps highlight that this option still uses the included CerticateManager to provide certificate management functionality. 

To use your own TLS certificate with CertificateManager

Opposed to putting an alternate load balancer in front of the deployment to handle certificate management.